care/facility/api/viewsets/bed.py

```diff
@@ -57,6 +59,28 @@ class BedViewSet(
     search_fields = ["name"]
     filterset_class = BedFilter
 
+    def create(self, request, *args, **kwargs):
```

Comment:
All of our core logic is usually placed in the serializer, can we move the current logic there?

Reply:
serializer.create should return single object so we would need to override and check the count and return appropriate response from viewset.create